packages/lab/src/utils/useIsViewportLargerThanBreakpoint.ts

```diff
@@ -9,8 +9,7 @@ export const useIsViewportLargerThanBreakpoint = (
 ): boolean => {
   const orderedBreakpoints = useOrderedBreakpoints();
   const index = orderedBreakpoints.indexOf(targetedBreakpoint);
-  const allPreviousBreakpoints = orderedBreakpoints.slice(0, index + 1);
-
   const currentBreakpoint = useCurrentBreakpoint();
-  return allPreviousBreakpoints.includes(currentBreakpoint);
```

Comment:
Not sure why checking to see if the current breakpoint is in the allPreviousBreakpoints is needed, so changed to a direct comparison of the index >= the current breakpoint